cnd/src/storage/db/tables.rs

```diff
@@ -508,14 +496,34 @@ pub struct InsertableBtcDaiOrder {
     pub order_id: i32,
     pub quantity: Text<Satoshis>,
     pub price: Text<Erc20Amount>,
+    pub open: Text<Satoshis>,
```

Comment:
📣 Because of the relation between `quantity` and the state quantities` I would suggest to use getters and checked setters instead.

Reply:
I started doing the checked setters, but after a quick discussion on how to achieve this in the best way with @thomaseizinger we decided to go for a simpler solution for now. 

Since the functions that manipulate the state (`set_to_settling`, `set_to_cancelled`) are in full control of the values, it should be fine with the tests for these functions in place. The tests ensure, that the complete amount is moved correctly for the current logic that does not include partial filling yet. 
Even when adding partial filling, the functions can apply these checks and the tests can be adapted accordingly.

Reply:
What I am trying to prevent is a future dev trying to change this values without using the setter.

Reply:
Note that testing the current logic is covered by `orderbook_dry.ts`. I think that is good enough for now. If the tests are not exhaustive enough please ping me.